### ISSUE TITLE
Bulb Capture with Canon EOS doesn't work correct

### ISSUE THREAD

**Issue by HKHH** [gphoto/gphoto2#362]
Hello,
an error seems to have crept into the canon eos remote function for the long exposure.

The command given on the gphoto homepage
gphoto2 --set-config eosremoterelease = Immediate --wait-event = 30s --set-config eosremoterelease = "Release Full" --wait-event-and-download = 2s
does not work. The shutter is not triggered in the B mode.

It works, however, if I put a LANG = GB in front of the command.

Tested with:
Ubuntu 20.04 (with german language setting)
gphoto2 2.5.23
libgphoto2 2.5.25
and Canon EOS 80D and Canon EOS 5D Mark IV

The following comes in the bulb mode:
gphoto2 --get-config shutterspeed
Label: Belichtungszeit                                                         
Readonly: 0
Type: RADIO
Current: Langzeitbelichtung
Choice: 0 Langzeitbelichtung
END

I think there is a problem with the translation function ...
bulb is translated in Langzeitbelichtung

**Comment by msmeissn** [comment]
yes, the examples currently assume "english" ... and will not work in german,. unless you specify LANG=C or LANG=GB or so
